.github/ISSUE_TEMPLATE/---01-bug-report.yml

```diff
@@ -30,20 +36,10 @@ body:
       description: A clear and concise description of what the bug is.
     validations:
       required: true
-  - type: textarea
+  - type: input
     attributes:
-      label: Steps to Reproduce
```

Comment:
I think allowing "steps to reproduce" is still useful, since it allows explaining what in particular is going wrong, though i suppose they can just add that content in the above section